### ISSUE TITLE
Haven't seen Overpass missions for some areas

### ISSUE THREAD

**Issue by Semigradsky** [kort/kort-native#49]
No any "Place without opening hours" or "Missing Levels" missions.
Android, v1.1

**Comment by sfkeller** [comment]
Thanks. But pls. document your issue so that anyone willing to contribute can reproduce the error or any missing data. Can you give any coordinates or street/city name where you observed this?

**Comment by Semigradsky** [comment]
I tried Minsk (Belarus), Moscow (Russia), Kyiv (Ukraine), Warsaw (Poland) and don't see these missions.

I see these missions for Bern (Switzerland), Paris (France).

**Comment by and01** [comment]
You are right. Some parts of the world miss Overpass missions. Since Overpass API heavily limits the requests, this workflow is rather cumbersome. So far only West Europe/UK have such missions.

Stay tuned for more areas to come.

**Comment by Semigradsky** [comment]
Maybe we can use user OSM oauth token for getting missions?

**Comment by and01** [comment]
The missions are being fetched on the backend and not on the client
